content/user/first_steps/yes_you_need_ssl.md

```diff
@@ -21,30 +21,35 @@ SSL encrypts communications between your customer's browser and your webserver.
 
 **DO I NEED SSL?**  
 
-If your site is collecting credit card info directly in a page inside your store (ie: not redirecting to a bank or payment gateway site to collect card info for payment) then YES you absolutely MUST use SSL to protect your customers' payment information.  
+Yes, you need SSL. 
 
-However, the best practice now is for all sites to run SSL all the time, 
-whether or not they are handling payment data. 
+Years ago, there was a line of thinking that if your site did payment collection offsite (e.g. PayPal Standard), you didn't really need SSL.  
 
-**HOW DOES ZEN CART IMPLEMENT MY SSL?**  
+Even back then, this was bad advice.  
+
+Here's why: without an SSL certificate, traffic to and from your website transits the Internet in clear plain text.  So anyone who's watching can see what's being sent. 
+
+- Without an SSL certificate, your admin credentials are transmitted in plain text, meaning anyone can see them and get access to your admin panel. 
+- Without an SSL certificate, your customer's personal information and purchase data are transmitted in plain text, meaning anyone who's watching can record these details. 
+
+Finally, SSL is considered a ranking factor by Google, and many browsers
+now warn customers not to purchase from sites that don't have SSL. 
 
-If you have SSL enabled in your hosting account (that's something you arrange with your hosting company directly), then you can tell Zen Cart to use your SSL URL, and then Zen Cart will automatically use that SSL URL when presenting pages dealing with sensitive information like login, account-creation, password changes, checkout, and even your admin pages.  
+See that ugly warning? 
 
-The modern way to use SSL is to have your entire site run as SSL, so the 
-`HTTP_SERVER` uses the `https` URL.  In the past,
-there was a separate HTTPS_SERVER for SSL, and 
-the `HTTP_SERVER` would serve pages over `http`.
-The reasoning was activities like browsing the catalog did not 
-require SSL, since there was no sensitive information exchanged. 
-This is no longer considered a good practice.
+**Don't let this be you!**
+
+<img src="/images/insecure_cart.png" alt="Insecure cart" width="50%" />
+
+**HOW DOES ZEN CART IMPLEMENT MY SSL?**  
 
-**BUT MY PAYMENTS HAPPEN OFFSITE.**  
+Arrange an SSL certificate with your hosting company, and then update your `configure.php` files to use your SSL address (the one starting with `https` rather than `http`). 
```

Comment:
Should this make it clear to put `https` URLs in BOTH the `HTTP_SERVER` and `HTTPS_SERVER` entries?

Reply:
Are you looking at the latest copy of this? It references each of the 5 or 6 defines that need updating. 

https://docs.zen-cart.com/user/first_steps/yes_you_need_ssl/

Reply:
Yes I had. 

I anticipate confusion in "the one starting with https rather than http", which precedes those 6 defines that all start with HTTP or HTTPS already. it's not clear that they should set the actual URL to begin with https in ALL of them (and not change the uppercase constant names)

Reply:
OK I see your point.  I added some detail and an example.